test/unit/algo/ptr_iterator.cpp

```diff
@@ -0,0 +1,57 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+
+#include "algo_test.hpp"
+
+#include <eve/algo/ptr_iterator.hpp>
+
+#include "iterator_conept_test.hpp"
+
+#include <array>
+#include <numeric>
+
+EVE_TEST_TYPES("Check unaligned_ptr_iterator", algo_test::selected_types)
+<typename T>(eve::as_<T>)
```

Comment:
Messed up the test name.
Will fix here if you have comments.
Otherwise - will do in the next patch.